src/game/client/components/countryflags.cpp

```diff
@@ -3,6 +3,7 @@
 #include "countryflags.h"
 
 #include <base/math.h>
+#include <base/log.h>
```

Comment:
```suggestion
#include <base/log.h>
#include <base/math.h>
```
clang-format nowadays wants sorted includes.